target/linux/ath79/dts/qca9533_comfast_cf-e110n.dts

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e110n", "qca,qca9533";
+	model = "COMFAST CF-E110N";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &wan;
+		led-upgrade = &rssilow;
```

Comment:
@rogerpueyo I'm sorry, that I've told you about this LED change. I did the same and [it was rejected](http://lists.infradead.org/pipermail/openwrt-devel/2018-December/015119.html), and the reason is, that it's not good idea to use RSSI LEDs for system states.

Reply:
No worries :)